src/moyo_list.erl

```diff
@@ -107,6 +108,13 @@ replace_if(PredFun, Value, List) -> replace_if_impl(PredFun, Value, List, []).
 -spec position(term(), list()) -> {ok, Position::pos_integer()} | error.
 position(Value, List) -> position_impl(Value, List, 1).
 
+%% @doc `List'内で最初に述語`PredFun'を満たす要素の位置を返す
+%%
+%% 該当する要素が存在しない場合は`error'が返される
```

Comment:
Could you mention that the returned index starts at 1, like in the parameter of `lists:nth/2`?

Reply:
I added explanation about that (although `pos_integer()` in the spec indicates to some extent that return values are 1-origin). Thanks for suggestion.